symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/properties/AppAuthProperties.java

```diff
@@ -0,0 +1,15 @@
+package com.symphony.bdk.app.spring.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Configuration Properties for enabling Application Authentication APIs.
+ */
+@Getter
+@Setter
+public class AppAuthProperties {
+
+  private Boolean enabled = true;
```

Comment:
javadoc is also very important on configuration properties fields (cf. https://www.baeldung.com/spring-boot-configuration-metadata#generating-metadata)